crates/trippy-tui/locales/app.yml

```diff
@@ -3,439 +3,549 @@ trippy:
   en: "trippy"
   fr: "trippy"
   tr: "trippy"
+  pt: "trippy"
 auto:
   en: "auto"
   fr: "automatique"
   tr: "otomatik"
+  pt: "automático"
 on:
   en: "on"
   fr: "activé"
   tr: "açık"
+  pt: "ligado"
 off:
   en: "off"
   fr: "désactivé"
   tr: "kapalı"
+  pt: "desligado"
 yes:
   en: "Yes"
   fr: "Oui"
   tr: "Evet"
+  pt: "Sim"
 no:
   en: "No"
   fr: "Non"
   tr: "Hayır"
+  pt: "Não"
 none:
   en: "none"
   fr: "aucun"
   tr: "hiçbiri"
+  pt: "nenhum"
 hidden:
   en: "Hidden"
   fr: "Caché"
   tr: "Gizli"
+  pt: "Oculto"
 flow:
   en: "flow"
   fr: "flux"
   tr: "akış"
+  pt: "fluxo"
 flows:
   en: "flows"
   fr: "flux"
   tr: "akışlar"
+  pt: "fluxos"
 target:
   en: "Target"
   fr: "Cible"
   tr: "Hedef"
+  pt: "Alvo"
 config:
   en: "Config"
   fr: "Config"
   tr: "Konfigurasyon"
+  pt: "Config"
```

Comment:
Config can be used as a short version, but actual word is "Configuração"

Reply:
Ok, will keep as "Config" then.